File: demo/src/java/exception/KoKoNotFoundId.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.ast;

public class KoKoNotFoundId extends RuntimeException {
	public KoKoNotFoundId(KoKoId id){
		super( String.format("*** Id '%s' is not defined! ***", id.getValue()) );
	}
	
}

File: KoKoslan/src/kotlin/exception/KoKoNotFoundId.kt
/*
    Andrey Arguedas Espinoza
    Daniela Armas Sanchez
    Michael Chen Wang
    Kimberly Olivas Delgado
*/

package kokoslan.kt.exception
import kokoslan.kt.ast.*

class KoKoNotFoundId(id: KoKoId) : RuntimeException("*** Id ${id.getValue()} is not defined! ***") { }